lib/Settings/licenses.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'dart:convert';
+
+class LicensesScreen extends StatefulWidget {
+  const LicensesScreen({Key? key}) : super(key: key);
+
+  @override
+  State<LicensesScreen> createState() => _LicensesScreenState();
+}
+
+class _LicensesScreenState extends State<LicensesScreen> {
+
+  late List data;
+  @override
+  void initState() {
+    // TODO: implement initState
+    super.initState();
+  }
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Licenses',style: TextStyle(color: Colors.black),),backgroundColor: Colors.white,foregroundColor: Colors.black,),
+      body: Center(
+        child: FutureBuilder(
+            future: DefaultAssetBundle.of(context)
+                .loadString('assets/licenses.json'),
+            builder: (context, snapshot) {
+              // Decode the JSON
+              data = json.decode(snapshot.data.toString());
+              return ListView.builder(scrollDirection: Axis.horizontal, shrinkWrap: true,
+          itemBuilder: (BuildContext context, int index) {
+            return Flexible(child:Card(
+              child: Padding(
+                padding: const EdgeInsets.only(
+                    top: 32, bottom: 32, left: 16, right: 16),
+                child: SingleChildScrollView(child:Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: <Widget>[
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        InkWell(
```

Comment:
Why are you using InkWell when there is no need for OnTap functionality?

Reply:
Removed!